src/components/main/main.js

```diff
@@ -110,6 +111,10 @@ app.component('main', {
             delete res.publicKey;
           }
 
+          this.notify.about(
```

Comment:
Yes, exactly. So you check the the balance is increased, then you pass the increased amount to `Notification.about`. My point is we should check and call if necessary, not always call and then perform if necessary.